src/main/java/uk/gov/hmcts/divorce/notification/CommonContent.java

```diff
@@ -55,8 +55,13 @@ public String getService(DivorceOrDissolution divorceOrDissolution) {
         return divorceOrDissolution.isDivorce() ? "divorce" : "civil partnership";
     }
 
-    public String getPartner(CaseData caseData) {
+    public String applicant1GetPartner(CaseData caseData) {
         return caseData.getDivorceOrDissolution().isDivorce()
             ? caseData.getApplicant2().getGender() == Gender.MALE ? "husband" : "wife" : "civil partner";
     }
+
+    public String applicant2GetPartner(CaseData caseData) {
```

Comment:
probably you can create a common method which can be used by both `applicant1GetPartner` and `applicant2GetPartner`